app/assets/javascripts/email_templates.js

```diff
@@ -0,0 +1,3 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+//= require tinymce
```

Comment:
It might be good to move this to webpack in the future, but if this works, then that's good for now.